cmd/cli/kubectl-kyverno/commands/test/test.go

```diff
@@ -98,6 +98,52 @@ func runTest(out io.Writer, testCase test.TestCase, registryAccess bool, auditWa
 	} else {
 		fmt.Fprintln(out, "  Applying", len(policies)+len(vaps), pluralize.Pluralize(len(policies)+len(vaps), "policy", "policies"), "to", len(uniques), pluralize.Pluralize(len(uniques), "resource", "resources"), "...")
 	}
+	policyToUnknownRules := make(map[string]string)
+	var policyerr string
+	for _, res := range testCase.Test.Results {
+		if res.IsValidatingAdmissionPolicy {
+			continue
+		}
+		var ns string
+		var policyExist bool
+		ruleExists := false
```

Comment:
Boolean variables are initialized to `false` by default. It is better to make it as follows:
```
var (
    policyExist bool
    ruleExist bool
)
```